tests/conftest.py

```diff
@@ -259,6 +275,80 @@ def mod_cylc_validate_cli(mod_capsys, mod_caplog):
     return _cylc_validate_cli(mod_capsys, mod_caplog)
 
 
+@pytest.fixture
+def rose_stem(test_dir, monkeypatch, request):
+    """The Rose Stem command.
+
+    Wraps the "rose_stem" async function for use in tests.
+
+    Cleans up afterwards if the test was successfull.
+    """
+    run_dir = test_dir / str(uuid4())[:4]
+
+    async def _inner(source_dir, cwd=None, **rose_stem_opts):
+        nonlocal monkeypatch, request, run_dir
+
+        # point rose-stem at the desired run directory
+        rose_stem_opts['no_run_name'] = True
+        rose_stem_opts['workflow_name'] = str(
+            run_dir.relative_to(CYLC_RUN_DIR)
+        )
+
+        # make it look like we're running the "rose stem" CLI
+        monkeypatch.setattr('sys.argv', ['stem'])
```

Comment:
Why is this not `['rose', 'stem']`?

Reply:
You tell me!

Reply:
Presumably the Rose root command swallows the first argument?